projects/README.md

```diff
@@ -0,0 +1,19 @@
+# End-of-project review checklist
+
++ The README
+  + is it obvious how to run the project
+  + is it obvious what it's for (Who? Why? How? What?)
+  + Have wireframes or diagrams been included?
+  + Any badges?
++ Issues
+  + How many are there?
+  + Any interesting ones?
+  + Have they been assigned?
+  + Have they been labelled?
+  + Have any issues been closed by anyone other than the person that raised them?
++ Commit history
+  + Do the commits tell a story?
++ Tests
++ Site
```

Comment:
Does this mean 'does the site exist or do I have to run this locally?' or 'You should have deployed to Heroku and included the link in your readme?'